backend/src/bundles/users/user.service.ts

```diff
@@ -18,7 +16,7 @@ class UserService implements IService {
     }
 
     public async find(
```

Comment:
I think you should create in user.repository.ts few methods to find user

Something like this:

` public async find(data: object): Promise<UserEntity | undefined> {
        const user = await this.userModel
            .query()
            .select()
            .where(data)
            .first();
        if (!user) {
            return undefined;
        }
        return UserEntity.initialize(user);
    }
`

and extend it in this way

`public async findByEmail(email: string): Promise<UserEntity | undefined> {
      return await this.find({ email });
 }`